activemq-osgi/src/main/resources/META-INF/spring.schemas

```diff
@@ -67,21 +67,32 @@ http\://activemq.apache.org/schema/core/activemq-core-5.15.9.xsd=activemq.xsd
 
 http\://camel.apache.org/schema/spring/camel-spring.xsd=camel-spring.xsd
 
-http\://www.springframework.org/schema/beans/spring-beans-2.0.xsd=org/springframework/beans/factory/xml/spring-beans-2.0.xsd
-http\://www.springframework.org/schema/beans/spring-beans-2.5.xsd=org/springframework/beans/factory/xml/spring-beans-2.5.xsd
-http\://www.springframework.org/schema/beans/spring-beans-3.0.xsd=org/springframework/beans/factory/xml/spring-beans-3.0.xsd
-http\://www.springframework.org/schema/beans/spring-beans-3.1.xsd=org/springframework/beans/factory/xml/spring-beans-3.1.xsd
-http\://www.springframework.org/schema/beans/spring-beans-3.2.xsd=org/springframework/beans/factory/xml/spring-beans-3.2.xsd
-http\://www.springframework.org/schema/beans/spring-beans.xsd=org/springframework/beans/factory/xml/spring-beans-3.2.xsd
-http\://www.springframework.org/schema/tool/spring-tool-2.0.xsd=org/springframework/beans/factory/xml/spring-tool-2.0.xsd
-http\://www.springframework.org/schema/tool/spring-tool-2.5.xsd=org/springframework/beans/factory/xml/spring-tool-2.5.xsd
-http\://www.springframework.org/schema/tool/spring-tool-3.0.xsd=org/springframework/beans/factory/xml/spring-tool-3.0.xsd
-http\://www.springframework.org/schema/tool/spring-tool-3.1.xsd=org/springframework/beans/factory/xml/spring-tool-3.1.xsd
-http\://www.springframework.org/schema/tool/spring-tool-3.2.xsd=org/springframework/beans/factory/xml/spring-tool-3.2.xsd
-http\://www.springframework.org/schema/tool/spring-tool.xsd=org/springframework/beans/factory/xml/spring-tool-3.2.xsd
-http\://www.springframework.org/schema/util/spring-util-2.0.xsd=org/springframework/beans/factory/xml/spring-util-2.0.xsd
-http\://www.springframework.org/schema/util/spring-util-2.5.xsd=org/springframework/beans/factory/xml/spring-util-2.5.xsd
-http\://www.springframework.org/schema/util/spring-util-3.0.xsd=org/springframework/beans/factory/xml/spring-util-3.0.xsd
-http\://www.springframework.org/schema/util/spring-util-3.1.xsd=org/springframework/beans/factory/xml/spring-util-3.1.xsd
-http\://www.springframework.org/schema/util/spring-util-3.2.xsd=org/springframework/beans/factory/xml/spring-util-3.2.xsd
-http\://www.springframework.org/schema/util/spring-util.xsd=org/springframework/beans/factory/xml/spring-util-3.2.xsd
+http\://www.springframework.org/schema/beans/spring-beans-2.0.xsd=org/springframework/beans/factory/xml/spring-beans.xsd
```

Comment:
That's actually not correct with Spring 4.3.24.RELEASE. For instance, the spring-beans xsd is located:

`org/springframework/beans/factory/xml/spring-beans-4.2.xsd`

Reply:
This is what I see in spring-beans:

```
jar tvf spring-beans-4.3.24.RELEASE.jar |grep -i xsd
 44429 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-4.2.xsd
  8612 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-4.3.xsd
 41645 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-3.0.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-4.2.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-4.0.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-4.3.xsd
  7663 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-2.0.xsd
 44429 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-4.3.xsd
  8092 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-3.2.xsd
  8092 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-3.1.xsd
 38848 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-2.0.xsd
 43993 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-3.2.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-3.1.xsd
  4345 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-3.0.xsd
  2835 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-2.5.xsd
 44429 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-4.1.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-3.2.xsd
  8612 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-4.1.xsd
  2208 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-2.0.xsd
 43193 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-4.0.xsd
  8612 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-4.2.xsd
  8092 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-4.0.xsd
  8068 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-3.0.xsd
  4341 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-tool-4.1.xsd
 43399 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-3.1.xsd
 42114 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-beans-2.5.xsd
  8068 Thu May 09 08:43:48 CEST 2019 org/springframework/beans/factory/xml/spring-util-2.5.xsd
```